site/en/docs/extensions/reference/scripting/index.md

```diff
@@ -200,29 +201,60 @@ If the resulting value of the script execution is a promise, Chrome will wait
 for the promise to settle and return the resulting value.
 
 ```js
+function getTabId() { ... }
 async function addIframe() {
-  const iframe = document.createElement('iframe');
-  const loadComplete = new Promise((resolve) => {
-    iframe.addEventListener('load', resolve);
-  });
-  iframe.src = 'https://example.com';
+  const iframe = document.createElement("iframe");
+  const loadComplete =
+      new Promise(resolve => iframe.addEventListener("load", resolve));
+  iframe.src = "https://example.com";
   document.body.appendChild(iframe);
   await loadComplete;
   return iframe.contentWindow.document.title;
 }
 
-const tabId = getTabId();
-chrome.scripting.executeScript(
-    {
-      target: {tabId: tabId, allFrames: true},
-      func: addIframe,
-    },
-    (injectionResults) => {
-      for (const frameResult of injectionResults)
-        console.log('Iframe Title: ' + frameResult.result);
+chrome.scripting
+    .executeScript({
+      target : {tabId : getTabId(), allFrames : true},
+      func : addIframe,
+    })
+    .then(injectionResults => {
+      for (const frameResult of injectionResults) {
+        const {frameId, result} = frameResult;
+        console.log(`Frame ${frameId} result:`, result);
+      }
     });
 ```
 
+## Examples
+
+### Unregister all dynamic content scripts
+
+The following snippet contains a function that unregisters all dynamic content
+scripts the extension has previously registered.
+
+```js
+async function unregisterAllDynamicContentScripts() {
+  try {
+    const scripts = await chrome.scripting.getRegisteredContentScripts();
+    const scriptIds = scripts.map(script => script.id);
+    return chrome.scripting.unregisterContentScripts(scriptIds);
+  } catch (error) {
+    const message = [
+      "An unexpected error occurred while",
+      "unregistering dynamic content scripts",
```

Comment:
```suggestion
      "unregistering dynamic content scripts.",
```

Reply:
I normally wouldn't have bothered with this, but you punctuated an error message farther down.